treeview.go

```diff
@@ -450,6 +450,12 @@ func (t *TreeView) SetSelectedFunc(handler func(node *TreeNode)) *TreeView {
 	return t
 }
 
+// GetSelectedFunc returns the function set with SetSelectedFunc() or nil
```

Comment:
Please use the Go link syntax here, i.e.

> ... set with [TreeView.SetSelectedFunc] or ...

You might have to change the line wrapping to adhere to the 80 character limit.

Reply:
Done.